src/Stryker.Core/Stryker.Core.UnitTest/Mutators/IntegerNegationMutatorTests.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.CodeAnalysis.CSharp;
+using Shouldly;
+using Stryker.Core.Mutators;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Xunit;
+
+namespace Stryker.Core.UnitTest.Mutators;
+
+public class IntegerNegationMutatorTests : TestBase
+{
+    [Fact]
```

Comment:
We have recently migrated all unit tests from xunit to mstest, sorry! Could you migrate your unit tests to mstest?